CommonTools/Utils/BuildFile.xml

```diff
@@ -1,3 +1,9 @@
+<use name="DataFormats/Common"/>
+<use name="DataFormats/Math"/>
+<use name="FWCore/ParameterSet"/>
+<use name="FWCore/Reflection"/>
+<use name="FWCore/SOA"/>
+<use name="FWCore/Version"/>
```

Comment:
this is apparently fake: it's included only in CommonTools/Utils/src/ExpressionEvaluator.cc, but the `getReleaseVersion` method is not used.

Reply:
Seems to be the case..